web/elm/tests/FlySuccessTests.elm

```diff
@@ -241,6 +271,19 @@ property query description assertion setup =
 
 
 
+-- subscription
+
+
+listensForTokenResponse : Property
+listensForTokenResponse setup =
+    test (setupDesc setup ++ ", listens for token response") <|
+        steps setup
+            >> Application.subscriptions
+            >> List.member Subscription.OnTokenSentToFly
+            >> Expect.equal True
```

Comment:
maybe a better 'list membership' expectation - this error message just says 'expected false to equal true' which is hard to understand in a hurry.